packages/flutter/lib/src/painting/flutter_logo.dart

```diff
@@ -38,53 +38,54 @@ enum FlutterLogoStyle {
 class FlutterLogoDecoration extends Decoration {
   /// Creates a decoration that knows how to paint Flutter's logo.
   ///
-  /// The [lightColor] and [darkColor] are used to fill the logo. The [style]
-  /// controls whether and where to draw the "Flutter" label. If one is shown,
-  /// the [textColor] controls the color of the label.
+  /// The [lightColor], [mediumColor], and [darkColor] are used to fill the
+  /// logo. The [style] controls whether and where to draw the "Flutter" label.
+  /// If one is shown, the [textColor] controls the color of the label.
   ///
-  /// The [lightColor], [darkColor], [textColor], [style], and [margin]
-  /// arguments must not be null.
+  /// The [lightColor], [mediumColor], [darkColor], [textColor], [style], and
+  /// [margin] arguments must not be null.
   const FlutterLogoDecoration({
-    this.lightColor = const Color(0xFF42A5F5), // Colors.blue[400]
-    this.darkColor = const Color(0xFF0D47A1), // Colors.blue[900]
-    this.textColor = const Color(0xFF616161),
+    this.lightColor = const Color(0xFF54C5F8),
```

Comment:
If we're disallowing color changes in the logo widget, why allow them here?

Reply:
No good reason. I'll remove it from here as well.